pytest_watcher/watcher.py

```diff
@@ -57,7 +67,11 @@ def _run_pytest(args) -> None:
     subprocess.run(["pytest", *args])
 
 
-def _parse_arguments(args: Sequence[str]) -> Tuple[Path, bool, float, Sequence[str]]:
+def _run_entrypoint(entrypoint) -> None:
```

Comment:
I believe that we should provide an ability to pass arguments to any runner. In that case having a separate function here should not be necessary